recipe/build.sh

```diff
@@ -44,3 +44,45 @@ FFLAGS="-fno-backtrace" cmake "${cmake_options[@]}"
 ninja all install
 
 popd
+
+
+if [ "${DFTBPLUS_TEST_BUILD}" != "True" ]; then
+  exit 0
+fi
```

Comment:
```suggestion
if [[ "${CONDA_BUILD_CROSS_COMPILATION:-0}" == "1" ]]; then
  exit 0
fi
```

Reply:
Very elegant, but how the hell did you know about this variable? :smile:  On https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html I could not find any hint about it...

Reply:
This one was introduced with the OSX/Arm migrations (https://conda-forge.org/docs/orga/minutes/2020-08-12.html?highlight=conda_build_cross_compilation), it is provided by the CTNG compiler activation (https://github.com/conda-forge/ctng-compiler-activation-feedstock).